### PR TITLE
#43 - hide sidebar starting at tablet displays, add teams page respon…

### PR DESCRIPTION
**Pull request by RubenSousaDinis**
…siveness

### CODE REVIEW

File: src/components/Player/index.js [hunk 1]
```diff
@@ -31,18 +37,34 @@ class Player extends Component {
...
+    this.handleResize();
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.handleResize);
+  }
+
+  handleResize = e => {
+    const toggled = window.innerWidth >= 992;
+    this.setState({toggled: toggled});
```
Comment (guivazcabral) [inline src/components/Player/index.js]:
```suggestion
    this.setState({ toggled });
```

File: src/assets/css/defaults/_layout.scss [hunk 2]
```diff
@@ -31,8 +31,13 @@
       display: inline-block;
       width: calc(100% - #{$content-margin} - 20px);
       padding-left: 0px;
```
Comment (guivazcabral) [inline src/assets/css/defaults/_layout.scss]:
```suggestion
       padding-left: 0;
```

File: src/components/Player/index.js [hunk 3]
```diff
@@ -31,18 +37,34 @@ class Player extends Component {
...
+  };
+
   render() {
     const {
       user,
       match
     } = this.props;
 
+    const contentClass = this.state.toggled ? 'toggled' : '';
```
Comment (guivazcabral) [inline src/components/Player/index.js]:
```suggestion
    const contentClass = this.state.toggled || '';
```

File: src/components/Team/index.js [hunk 4]
```diff
@@ -120,6 +120,7 @@ class Team extends Component {
           user={user}
           links={links}
           shortcutLinks={shortcutLinks}
+          toggled={true}
```
Comment (guivazcabral) [inline src/components/Team/index.js]:
```suggestion
          toggled
```

when a prop is a boolean, don't need to set a value for it. a `<Component toggled>` will have a prop `toggled` as `true`. So, in the end `<Component toggled>` and `<Component toggled={true}>` are the same thing

Comment (RubenSousaDinis) [reply src/components/Team/index.js]: Cool, thanks!

**Pull request merged**